app/services/user_service/sync_with_loops.rb

```diff
@@ -3,7 +3,7 @@
 module UserService
   class SyncWithLoops
     def initialize(user_id:, queue: Limiter::RateQueue.new(2, interval: 1), new_user: false)
-      @user = User.find(user_id)
+      @user = User.find(user_id).includes(:events)
```

Comment:
```suggestion
      @user = User.includes(:events).find(user_id)
```

Reply:
Whoops, thanks. We can drop the includes entirely here. Since it's just one record, there's no advantage eager loading it